Update Honeypot Intro (#35)
epicweb-dev/web-forms: exercises/06.honeypot/README.mdx

To implement a honeypot field, you'd typically add an input field to your form
and then hide it using CSS. It's crucial that this field is hidden in a way

> that's not detectable by human. It's also a good idea to give the honeypot field

Reviewer: Yeah, you're right. This is confusing. How about we reword it instead?

```suggestion
that humans can't use, but a bot will. It's also a good idea to give the honeypot field
```
